### ISSUE TITLE
sequencer bar grid lines don't line up musically.

### ISSUE THREAD

**Issue by Lykdraft** [kmatheussen/radium#1205]
I noticed a very odd design choice.

The background grid in the sequencer looks very weird when zoomed out a bit. I see strong lines at the beginning of bar 4,7,10,13,16 and so on.

Usually in every DAW I know you have those lines at 5,9,12,17 and so on. 

This is weird, because you usually need to see when a new musical measure starts, (i.e.: in 4/4 this is after 4 bars have played, thus on 5 you have a visual aid, not on 4, which actually means that a visual aid is at an odd position since only 3 bars have played at that point in time...)

Simple example: When we have 2 blocks with a 32 line lenght 
in the sequencer, a big gridline cuts through the middle of the second block (at position 4) which doesn't make sense. It gives an odd impression. The Gridline should be on 5, which indicates that at that point 4 bars have played.

I hope this makes sense.

Best,
tL.

(Obviously you could expand on that and let the background grid adapt to the time signature for easier visual guide/alignment for other timesignatures than 4/4)

**Comment by Lykdraft** [comment]
Ah, I see. This behavior is zoom dependent. 
Looks like this is only happening at an "inbetween" zoom setting. I can see the lines snap out of place when zooming out and when zooming out even more, they snap back in place. Weird.

**Comment by Lykdraft** [comment]
That said, maybe recallable zoomsettings might be a good addition as well.

**Comment by kmatheussen** [comment]
Yeah, it sounds less confusing if the background grid was always placed at
the same bar numbers.
For instance, either
* 1,2,3,4,5
* 1,3,5,7,9
* 1,5,9,13,17
* 1,9,17,25,33
* 1,17,33,49,65
etc.

Currently the program tries to spread the number of grid lines
approximately so that not too many or too few lines are shown.
Also, it shouldn't make a difference whether signature is 3/4,4/4,3/8,5/4,
etc.

**Comment by Lykdraft** [comment]
> Also, it shouldn't make a difference whether signature is 3/4,4/4,3/8,5/4,

True. Of course it shouldn't. Was late yesterday.
